terasoluna-gfw-parent/pom.xml

```diff
@@ -346,18 +353,13 @@
       </dependency>
       <!-- == End MyBatis == -->
 
-      <!-- == Begin Dozer== -->
+      <!-- == Begin MapStruct== -->
       <dependency>
-        <groupId>com.github.dozermapper</groupId>
-        <artifactId>dozer-core</artifactId>
-        <version>${dozer.version}</version>
+        <groupId>org.mapstruct</groupId>
+        <artifactId>mapstruct</artifactId>
+        <version>${mapstruct.version}</version>
       </dependency>
-      <dependency>
-        <groupId>com.github.dozermapper</groupId>
-        <artifactId>dozer-spring4</artifactId>
-        <version>${dozer.version}</version>
-      </dependency>
-      <!-- == End Dozer== -->
+      <!-- == End MapStruct== -->
```

Comment:
https://mapstruct.org/faq/#Can-I-use-MapStruct-together-with-Project-Lombok

You can provide also [lombok-mapstruct-binding](https://search.maven.org/artifact/org.projectlombok/lombok-mapstruct-binding).

Reply:
005161d296661ae8f8fab0ed0d5aadb0836a246d
Added Lombok dependency to `maven-compiler-plugin`. Also added `mapstruct.defaultComponentModel=sprin` to allow autowiring by default.

Reply:
> Added Lombok dependency to maven-compiler-plugin.

I don't think you should put the Lombok and MapStruct processors as all of default setting.
Whether or not use Lombok or MapStruct should be completely up to users.

> mapstruct.defaultComponentModel=sprin to allow autowiring by default.

You can set this to either.
However, you need to be careful not to misunderstand TERASOLUNA users
that MapStruct is automatically injected "for some unknown reason".

You can see the common library policy as follows:

https://github.com/terasolunaorg/terasoluna-gfw#terasoluna-server-framework-for-java-5x-common-library

> The common library of TERASOLUNA Server Framework for Java (5.x) is a library of useful and **unobtrusive** common functionalities.

Thanks.

Reply:
Thanks.

Since Dozer, which existed as a BeanMapper, could be used simply by loading terasoluna-gfw, I was thinking of making mapstruct usable without configuration as well. Also, considering the use with lombok, we thought it would be better to set it all up together from the beginning.

However, since the user sets whether to use lombok or not on the empty project side, we would like to configure the plugin on the empty project side so that the user can choose whether to use lombok or not.

Reply:
https://github.com/terasolunaorg/terasoluna-gfw/pull/1184/commits/b69e97ac01deda21cefaa62fd84cea44804a261a
Deleted mapstruct dependencies from `maven compiler plugin`

> It is not to be corrected by this ISSUE.

https://github.com/terasolunaorg/terasoluna-gfw/pull/1184/commits/6d76cf03a6716b42275c3e54b7bd2a388575117b
Deleted unnecessary changes.

Reply:
You forgot the first point...
Who manage the version of lombok-mapstruct-binding?

Reply:
In order to configure the plugin in the blank project, I was thinking of setting the properties on the blank project side, but since there is a possibility that there will be differences with the version provided by gfw, I will define them in gfw-parent.
As for lombok, it depends on boot-dependencies, but the properties are not available, so I will redefine the properties in gfw-parent.
https://github.com/terasolunaorg/terasoluna-gfw/pull/1188